packages/fxa-content-server/app/scripts/templates/partial/firefox-family-services.mustache

```diff
@@ -5,10 +5,18 @@
       <span class="logo logo-firefox-browser">{{#t}}Firefox browser{{/t}}</span>
       {{#t}}Sync your bookmarks, passwords, and more on every device.{{/t}}
     </li>
-    <li class="firefox-service service-lockwise">
-      <span class="logo logo-lockwise">{{#t}}Firefox Lockwise{{/t}}</span>
-      {{#unsafeTranslate}}Securely access passwords saved in Firefox anywhere &mdash; even outside of the browser.{{/unsafeTranslate}}
-    </li>
+    {{#isInPocketMigration}}
+      <li class="firefox-service service-pocket">
+        <span class="logo logo-pocket">{{#t}}Pocket{{/t}}</span>
+        {{#unsafeTranslate}}Save and discover the best stories from accross the web.{{/unsafeTranslate}}
```

Comment:
I don't think this needs to be in an unsafeTranslate since it doesn't contain html.